src/mantine-core/src/Floating/FloatingArrow/FloatingArrow.tsx

```diff
@@ -1,36 +1,19 @@
 import React, { forwardRef } from 'react';
-import { useMantineTheme } from '@mantine/styles';
 import { getArrowPositionStyles } from './get-arrow-position-styles';
 import { FloatingPosition } from '../types';
 
 interface FloatingArrowProps extends React.ComponentPropsWithoutRef<'div'> {
   withBorder: boolean;
   position: FloatingPosition;
   arrowSize: number;
-  arrowOffset: number;
   arrowRadius: number;
   arrowX: number;
   arrowY: number;
   visible: boolean;
 }
 
 export const FloatingArrow = forwardRef<HTMLDivElement, FloatingArrowProps>(
-  (
-    {
-      withBorder,
-      position,
-      arrowSize,
-      arrowOffset,
-      arrowRadius,
-      visible,
-      arrowX,
-      arrowY,
-      ...others
-    },
-    ref
-  ) => {
-    const theme = useMantineTheme();
```

Comment:
We don't need the theme at all anymore since the RTL direction is already handled by Floating UI